package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "@yext/answers-headless",
   "version": "1.2.0",
-  "description": "",
+  "description": "A library for powering UI components for Yext Search integrations",
   "author": "slapshot@yext.com",
   "license": "ISC",
   "main": "./dist/commonjs/src/index.js",
   "module": "./dist/esm/src/index.js",
+  "keywords": [
+    "Search",
+    "Yext",
```

Comment:
I would change redux and state management to lowercase. npm search is case insensitive so it probably doesn't matter but it's what I'm more used to seeing
see https://www.npmjs.com/package/redux